awx/main/tasks.py

```diff
@@ -1279,6 +1279,8 @@ def status_handler(self, status_data, runner_config):
                 if k in job_env:
                     job_env[k] = v
             self.instance = self.update_model(self.instance.pk, job_args=json.dumps(runner_config.command), job_cwd=runner_config.cwd, job_env=job_env)
+        elif status_data['status'] == 'error':
+            self.instance = self.update_model(self.instance.pk, result_traceback=status_data['result_traceback'])
```

Comment:
but will the "result_traceback" key always be present??

Reply:
Fair enough question. I'll add some guardrails.